plugin/trino-exchange-filesystem/pom.xml

```diff
@@ -14,8 +14,8 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <awsjavasdk.version>2.17.151</awsjavasdk.version>
-        <azurejavasdk.version>1.1.1</azurejavasdk.version>
+        <awsjavasdk.version>2.20.47</awsjavasdk.version>
+        <azurejavasdk.version>1.2.11</azurejavasdk.version>
```

Comment:
should we not match the versions here with those in root pom to avoid this in first place? Any reason why they use different versions?

Reply:
I think regarding AWS we have `1.*` in root pom, right?

Reply:
no, we use 1.x everywhere (as possible).

Reply:
currently it's at 1.12.145 in root.